media/css/custom.scss

```diff
@@ -583,3 +583,40 @@ summary {
     width: 87px;
   }
 }
+
+.council-item {
+  padding-bottom: 25px;
+
+  img {
+    width: 290px;
+  }
+
+  .img-centered {
+    display: block;
+    margin: 0 auto;
+  }
+}
+
+.council-wrapper {
+  div {
+    margin-bottom: 20px;
+    text-align: center;
+  }
+
+  span {
+    font-size: 24px;
+    color: white;
```

Comment:
We have css variables declared in `media/css/_variables.scss`, so you should reference `$white` instead.